pyartcd/tests/pipelines/test_prepare_release_konflux.py

```diff
@@ -587,19 +585,6 @@ def get_snapshot(builds):
         self.assertEqual(created_shipments_arg["extras"], mock_shipment_extras_create)
         self.assertEqual(mock_create_shipment_mr.call_args[0][1], "prod")
 
-        # assert that build-data PR was created with the right config
```

Comment:
I'm not sure why this assertion is not being valid now? 
If the code path changed, then i would want a new assertion added for the build data pr content.
And if you can add the release jira key as well.